eng/templates/official/jobs/build-artifacts.yml

```diff
@@ -0,0 +1,35 @@
+jobs:
+  - job: "Build"
+    displayName: 'Build Python Extensions'
+
+    strategy:
+      matrix:
+        base_extension:
+          EXTENSION_DIRECTORY: 'azurefunctions-extensions-base'
+          EXTENSION_NAME: 'Base'
+        blob_extension:
+          EXTENSION_DIRECTORY: 'azurefunctions-extensions-bindings-blob'
+          EXTENSION_NAME: 'Blob'
+        fastapi_extension:
+          EXTENSION_DIRECTORY: 'azurefunctions-extensions-http-fastapi'
+          EXTENSION_NAME: 'FastAPI'
+    templateContext:
+      outputParentDirectory: $(Build.ArtifactStagingDirectory)
+      outputs:
+        - output: pipelineArtifact
+          targetPath: $(Build.SourcesDirectory)
+          artifactName: "$(EXTENSION_DIRECTORY)"
+
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.11'
```

Comment:
build the extension also for other python versions, so version compaitibilities can be tested?

Reply:
Discussed offline. Going to stick with the latest stable supported version. Any issues with compatibility will be caught in the worker tests.